src/core/subject/requisits/requisits.enum.ts

```diff
@@ -0,0 +1,6 @@
+export enum Requisits {
```

Comment:
Deixar um nome mais explícito.
```suggestion
export enum SubjectRequisit {
```